frontend/packages/core/src/timeago.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import type { ReactTimeagoProps as TimeAgoProps } from "react-timeago";
+import ReactTimeago from "react-timeago";
+
+interface EventTimeProps extends Pick<TimeAgoProps, "date"> {
+  onClick?: () => void;
+  short?: boolean;
+  live?: boolean;
```

Comment:
what does `live` do? i toggled it and didn't see a change

Reply:
It really only works if you're in the seconds (put in a new timestamp of now from epoch converter and toggle it then). It just auto increments the time for you instead of being static.

Yeah I'll add a comment

Reply:
maybe we should call it static and invert the defaults / logic

Reply:
Shouldn't this also work in the minutes and hours if you have the page open long enough?

Reply:
It would, its just easiest to see from a testing perspective in seconds

Reply:
Static is a reserved keyword, so that seems to not work